hedera-mirror-rest/viewmodel/networkNodeViewModel.js

```diff
@@ -0,0 +1,54 @@
+/*-
+ * ‌
+ * Hedera Mirror Node
+ * ​
+ * Copyright (C) 2019 - 2022 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+'use strict';
+
+const AddressBookServiceEndpointViewModel = require('./addressBookServiceEndpointViewModel');
+const EntityId = require('../entityId');
+const utils = require('../utils');
+
+/**
+ * Network node view model
+ */
+class NetworkNodeViewModel {
+  /**
+   * Constructs network node view model
+   *
+   * @param {AddressBook} addressBook
+   */
+  constructor(networkNode) {
+    this.description = networkNode.addressBookEntry.description;
+    this.file_id = EntityId.parse(networkNode.addressBook.fileId).toString();
+    this.memo = networkNode.addressBookEntry.memo;
+    this.node_id = Number(networkNode.addressBookEntry.nodeId);
+    this.node_account_id = EntityId.parse(networkNode.addressBookEntry.nodeAccountId).toString();
+    this.node_cert_hash = utils.encodeBase64(networkNode.addressBookEntry.nodeCertHash);
```

Comment:
Should be hex as in the HIP and since hashes are generally encoded in hex. Note that the underlying bytea is already hex encoded. Pseudo code:

```suggestion
    this.node_cert_hash = '0x' + decodeHex(networkNode.addressBookEntry.nodeCertHash).toString();
```

Please confirm the output matches the node cert hash in the [docs](https://docs.hedera.com/guides/mainnet/mainnet-nodes).